### DISCUSSION TITLE
how can i list user's linked-accounts by postman

### DISCUSSION THREAD

**Discussion by linlinwen** [keycloak/keycloak#12200] [Q&A] [answered]
I want to list user's linked-accounts by postman, but return with error message: "RESTEASY003210: Could not find resource for full path: http://localhost:8080/realms/smsb/account/linked-accounts".
![image](https://user-images.githubusercontent.com/67309521/170462941-36858518-16f2-4813-84ea-a67264313e11.png)

But actually, i can get it in keycloak account page:
![image](https://user-images.githubusercontent.com/67309521/170462880-7837e080-6c0c-4ed6-83a6-775e4e89d683.png)

And i saw the test case using the same url:
![image](https://user-images.githubusercontent.com/67309521/170463071-3284e5b3-659b-42df-9f55-727b2fea95b0.png)

**Comment by linlinwen** [comment]
I has tested the url "http://localhost:8080/realms/smsb/account/linked-accounts" and "http://localhost:8080/auth/realms/smsb/account/linked-accounts", but return the same error.

**Comment by linlinwen** [comment]
I use keycloak in docker, the version is quay.io/keycloak/keycloak:17.0.0

**Comment by abstractj** [comment, marked as answer]
@linlinwen I tried to reproduce the issues you mentioned using the latest releases (18.0.0) and I was able to send request to the `linked-accounts` endpoint.

If you would like, I added my Postman workspace here (https://gist.github.com/abstractj/6809fa2d9b0dc1ee87cbf95541cc84fa) with instructions.

I hope it helps.

**Reply by linlinwen** [reply]
@abstractj Hi abstractj, I access the linked-accounts endpoint with your help. And I found the reason was I didn't declare the content-type header, but as the pic show postman default content-type is application/json. I guess it maybe is the bug of postman? 
![image](https://user-images.githubusercontent.com/67309521/170830819-bcb9c57c-404a-4f54-8e8f-4de1af153a83.png)

And thanks for your kind help.

**Comment by abstractj** [comment]
@linlinwen to be honest, I don't know if it's Postman bug, but I'm glad that is working for you now.
